dkarlovi/symfony-flex-docker/0.1/manifest.json

```diff
@@ -0,0 +1,18 @@
+{
+    "copy-from-package": {
+        ".infra/": ".infra",
+        ".dockerignore": ".dockerignore",
+        "docker-compose.yaml": "docker-compose.yaml"
+    },
+    "env": {
+        "COMPOSE_PROJECT_NAME": "symfony",
+        "CACHE_HOST": "cache",
+        "CACHE_PORT": "6379",
+        "DATABASE_HOST": "db",
+        "DATABASE_PORT": "3306",
+        "DATABASE_NAME": "symfony",
+        "DATABASE_USER": "symfony",
+        "DATABASE_PASSWORD": "%generate(secret)%",
+        "DATABASE_ROOT_PASSWORD": "%generate(secret)%"
```

Comment:
It's tougher to discover for the user, IMO. This way, you can setup your database GUI tool just by opening `.env`. If you want, you can also change the password and have a place to do it.